client/my-sites/domains/domain-management/subpage-wrapper/style.scss

```diff
@@ -6,6 +7,11 @@
 	flex-direction: column;
 	height: 100%;
 
+    .button:not(.is-borderless) {
```

Comment:
This block overrides other button styles, see the example:

<img width="641" alt="Screenshot 2024-12-25 at 14 49 41" src="https://github.com/user-attachments/assets/90c70c60-eacd-4bf2-8a61-ecce664f4790" />

I would rather keep the default button style than override it.